### PR TITLE
Add RFmxInstr enums and attributes

### PR DESCRIPTION
**Pull request by astarche**
### What does this Pull Request accomplish?

Adds enums and attributes to the RFmxInstr service.

### Why should this Pull Request be merged?

Improve usability of the RFmxInstr service by adding the enums and attributes defined in the C API.

Fixes [AB#1781494](https://ni.visualstudio.com/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/1781494).

### What testing has been done?

Ran and passed all RFmx tests including new tests.

Some additional free form testing with RFmxInstr attributes.

### CODE REVIEW

File: source/tests/system/nirfmxinstr_driver_api_tests.cpp [hunk 1]
```diff
@@ -132,7 +143,30 @@ TEST_F(NiRFmxInstrDriverApiTests, NoActiveList_GetListNames_ReturnsEmptyLists)
...
+  initiate_to_enter_committed_state(session);
+  const auto response = client::get_attribute_i32(stub(), session, "", NiRFmxInstrAttribute::NIRFMXINSTR_ATTRIBUTE_TUNING_SPEED);
+
+  EXPECT_SUCCESS(session, response);
+  EXPECT_EQ(response.attr_val(), NiRFmxInstrInt32AttributeValues::NIRFMXINSTR_INT32_TUNING_SPEED_FAST);
+}
+
+TEST_F(NiRFmxInstrDriverApiTests, SetAndGetFrequencyReferenceSource_ReturnsFrequencyReferenceSource)
+{
+  constexpr auto MIXER_LEVEL = 5.5e7;
```
Comment (astarche) [inline source/tests/system/nirfmxinstr_driver_api_tests.cpp:159]: TODO Remove bogus constant

File: source/codegen/metadata/nirfmxinstr/enums.py [hunk 2]
```diff
@@ -1,41 +1,244 @@
 enums = {
```
Comment (astarche) [inline source/codegen/metadata/nirfmxinstr/enums.py:1]: TODO sync with scrapigen PR.

File: source/codegen/metadata/nirfmxinstr/enums_addon.py [hunk 3]
```diff
@@ -4,3 +4,6 @@
 enums_override_metadata = {
 }
 
+enums_validation_suppressions = {
+    "FrequencyReferenceExportedTerminal": ["ENUMS_SHOULD_NOT_HAVE_DUPLICATE_VALUES"]
```
Comment (reckenro) [inline source/codegen/metadata/nirfmxinstr/enums_addon.py:8]: What is this bit about? I might be missing something but I don't see this 'enums_validation_suppressions' string referenced anywhere except the `__init__.py` and `enums_addon.py` for a few drivers.

**Review by reckenro — APPROVED** [review]

**Review by mgumble — APPROVED** [review]

File: source/codegen/metadata/nirfmxinstr/enums_addon.py (hunk 3 above)
Comment (astarche) [reply source/codegen/metadata/nirfmxinstr/enums_addon.py:8]: Yeah! This was tricky to track down.

1. There's a rule in metadata_validation to require no duplicate enum values.
2. There's a way to override that rule.
3. BUT how does it work!?

I'm guessing that the bit you missed was the same one that took me forever to find: [rule_is_suppressed](https://github.com/ni/grpc-device/blob/d6fddbe4f42eb9f4c36fd3d037385cf7dc1a6088/source/codegen/metadata_validation.py#L155).
 So basically the "enums" prefix means it applies to the `enums.py` metadata file
